circle.yml

```diff
@@ -0,0 +1,15 @@
+machine:
+  pre:
+    - curl -sSL https://s3.amazonaws.com/circle-downloads/install-circleci-docker.sh | bash -s -- 1.10.0
+  services:
+    - docker
+
+dependencies:
+  override:
+    - docker info
+    - docker pull slicer/slicer-build-deps
+
+test:
+  override:
+    - "git diff --name-only master | grep -q SuperBuild > /dev/null || echo 'Notice: TravisCI does not build changes to Slicer dependencies'"
```

Comment:
Yes, it was supposed to have a non-zero status code and cause the build to fail -- please check this follow-up.

  https://github.com/Slicer/Slicer/pull/488

I had also tried TravisCI and missed the name update. TravisCI would time out at its fixed 40 minute limit.